www/about/policies/terms-of-service.md

```diff
@@ -97,23 +98,24 @@ payments is solely between those parties.
 
 1. Terms for Givers
 
-  1. When you make a payment to a Project, it is a payment to fund the Work
-     described in the Project's profile. You understand and acknowledge that
-your payment does not entitle you personally to receive any goods or services
-from the Project, unless the Project explicitly offers those things to Givers.
+  1. When you make a payment to a Project Owner, it is a payment to fund the
+     Work described in the Project's profile. You understand and acknowledge
+that your payment does not entitle you personally to receive any goods or
+services from the Project Owner, unless the Project Owner explicitly offers
```

Comment:
There is currently no mechanism for the Project Owner to know which ~users are actually giving to a Project (https://github.com/gratipay/gratipay.com/issues/236 would change this), which would be required for Project Owners to offer work to individual Givers according to what they give. I would take out the "unless" clause entirely, unless @whit537 thinks we're heading in that direction (in which case, whatever you say, @tieguy!).